vitrina/orgs/admin.py

```diff
@@ -70,7 +70,7 @@ def has_delete_permission(self, request: HttpRequest, obj: object | None = None)
         return request.user.is_staff or request.user.is_superuser
 
 
-class OrganizationAdmin(VersionAdmin, TreeAdmin):
+class OrganizationAdmin(TreeAdmin, RevisionCommentVersionAdmin):
```

Comment:
I'd be careful mixing the base classes around and leave it as was:


```suggestion
class OrganizationAdmin(RevisionCommentVersionAdmin, TreeAdmin):
```

Unless you are completely sure, that no inheritance-based problems arise from that

Reply:
https://github.com/atviriduomenys/katalogas/pull/2179/commits/47977a3c6845d3bab3e911617ad31954609103e4